src/AWS.Deploy.Common/Recipes/Validation/OptionSettingItemValidators/ComparisonValidator.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\r
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Threading.Tasks;
+
+namespace AWS.Deploy.Common.Recipes.Validation
+{
+    public enum ComparisonValidatorOperation
+    {
+        GreaterThan
+    }
+
+    /// <summary>
+    /// The validator is typically used with OptionSettingItems which have a numeric type.
+    /// The validator requires two configuration options to be specific, the Operation and the SettingId.
+    /// The validator checks if the set value of the OptionSettingItem satisfies the comparison operation with SettingId.
+    /// </summary>
+    public class ComparisonValidator : IOptionSettingItemValidator
```

Comment:
This seems very similar to [MinMaxConstraintValidator.cs](https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.Common/Recipes/Validation/RecipeValidators/MinMaxConstraintValidator.cs).

1. Could we reuse the existing validator here?
2. Or if this new one is preferred, could you also replace MinMax?

Reply:
I removed the MinMaxConstraint validator as it can be replaced by the new one.